syncthing.fc

```diff
@@ -0,0 +1,8 @@
+# binary
+/usr/bin/syncthing                      -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+/usr/local/bin/syncthing                -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+HOME_DIR/.local/bin/syncthing           -- gen_context(system_u:object_r:syncthing_exec_t,s0)
```

Comment:
This is not desired, and probably not needed. The transition happens on /usr/bin/synchting